role/role.go

```diff
@@ -18,29 +18,47 @@ package role
 import (
 	"strings"
 
+	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/pkg/errors"
-	"github.com/yahoojapan/athenz-policy-updater/pubkey"
+	"github.com/yahoojapan/athenz-authorizer/pubkey"
 )
 
-// RoleTokenParser represents the role token parser interface.
-type RoleTokenParser interface {
-	ParseAndValidateRoleToken(tok string) (*RoleToken, error)
+// Role represents the role token parser interface.
+type Role interface {
+	ParseAndValidateRoleToken(tok string) (*Token, error)
+	ParseAndValidateRoleJWT(tok string) (*Claim, error)
 }
 
 type rtp struct {
 	pkp pubkey.Provider
 }
 
-// NewRoleTokenParser returns the RoleTokenParser instance.
-func NewRoleTokenParser(prov pubkey.Provider) RoleTokenParser {
+// New returns the Role instance.
+func New(prov pubkey.Provider) Role {
 	return &rtp{
 		pkp: prov,
 	}
 }
 
+func (r *rtp) ParseAndValidateRoleJWT(tok string) (*Claim, error) {
```

Comment:
need unit test?

Reply:
I think we should, but I have no time can add UT later?